NEWS.md

```diff
@@ -1,7 +1,23 @@
-## 6.1.0 - Unreleased
+## 8.0.0 - Unreleased
 
-## 6.0.0 - Released
+## 7.0.0 - Released
+The primary focus of this release was to implement CRUD API for managing Teams (Units, Assignments, Memberships) and business logic improvements.
+
+[Full Changelog](https://github.com/folio-org/mod-orders/compare/v6.0.0...v7.0.0)
 
+### Stories
+ * [MODORDERS-277](https://issues.folio.org/browse/MODORDERS-277)	- Update inventory interface version
```

Comment:
nitpick: To be consistent with NEWS.md in other projects, remove one extra space between `)` and `-`

Reply:
@javalkar-varun, noted. Thanks!